### PR TITLE
Lab5 Peer Review

### PR DESCRIPTION
**Pull request by williamdolsen**

### CODE REVIEW

**Comment by jenniv27** [comment]
Introductory writing:

The Higgs Boson section
- "four elections" I believe you mean electrons here.
- I think you should elaborate on the following part: "within our dataset we are trying to detect the Higgs Boson via it's decay to a bottom quark-antiquark pair." How do you know we are looking at this specific decay? I might have missed that.
- Do you think we need to cite our sources in this report? And whether it should be formatted in a specific way?

Experimental details section
- when defining your jet, I think it would be helpful to specify that R is the radius (not just the size) and that the distance is from some specific particle at the center of the cone. otherwise the jet, as you define it, has a size which is somewhat not defined well and in an unknown direction
- "Once we have defined a jet, we can use the the 4-momentum of the jet - the sum of all of the 4-momenta of the particles in the jet - to determine a possible value for the 4-momentum of the particle that created the jet initially, through decay. " In this section I think you should explicitly mention the property of "conservation of 4-momentum"
- "the better will will represent the data however there is a point where we will begin to become bias towards our data which is something to be avoided." I'm not sure what you mean we would become biased towards our data? Can you explain this?

Great logical order of explanation for the introduction overall!

**Comment by jenniv27** [comment]
Experimental Data:
- What is "boost invariant"?
- What is an "energy correlation function"?

Data Observations: 
- "We see that the majority of the Higgs Bosons have a $pT$ of around 1100 $\pm$ 25 however there is also a large amount of data that lies outside of that range." I would not claim that the "majority" of the data is in this range. It seems like only ~1/3 of the data would be in that range. The width of the distribution is quite large
- "We also see that jets with a $\eta$ around $\frac{\pi}{2}$ are favored." Do you mean \theta here?
- For ee3, you write "We see that the QCD distribution is defined on the range 0 to about 0.025 with a peak around 0.00025" how did you come to find such a precise peak value, instead of just saying about 0? If you did a calculation, you might want to include that code here.
- For t3, t21, and t32 I think you are confusing your skew directions: 
image

**Comment by jenniv27** [comment]
Observations summary:
- "We found that ee2, ee3, and d2 all provide discrimination power in the data set and these features are correlated. Specifically, d2 is correlated with ee2 and d2 is correlated with ee3 because ..." You introduced this formula before making observations of the data so I'm not sure you can say you "found" that these features are all correlated. Same with t21 and t32 ratios. However perhaps what you can say is "I have confirmed that these variables are indeed correlated"

Base discovery significance:
- "What is the significance of my signal of 2050 provided the QCD background - a Poisson distribution with parameter 2000?" I would reword this slightly as "What is the significance of my signal of 2050 (jets) provided the QCD background (is Poisson distributed) with (lambda parameter) of 2000 (jets)?"
- After you calculate the number of jets (not particles) required to see a 5 sigma event you should make comment on the comparison of this value vs the expected (2050)

Discovery sensitivity optimization:
- Great job! Nice report overall

**Comment by williamdolsen** [comment]
@AlexSchuy
